matomo-proxy.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+define('MATOMO_PROXY_FROM_ENDPOINT', 1);
+
+$path = 'index.php';
+
+$SUPPORTED_METHODS = [
+    'CoreAdminHome.optOut',
+];
+
+$allParams = $_GET + $_POST;
+
+if (!in_array("{$allParams['module']}.{$allParams['action']}", $SUPPORTED_METHODS)) {
+    http_response_code(404);
```

Comment:
After trying around a bit I couldn't find a way to circumvent this check.
But I noticed that a simple request to `matomo-proxy.php` causes an error (because `module` and `action` are not set), which is a bit ugly.

And the whole security depends on the order of `$_GET + $_POST` and the next person to edit this to know that php adds arrays from right to left overwriting `POST` by `GET`.

If someone switches this around accidentally, all Matomo modules can be accessed:
```
curl --data "module=CoreAdminHome&action=optOut" "http://localhost/tracker-proxy/matomo-proxy.php?module=Login&idSite=1&period=day&date=yesterday&action=
```
So I'd at least add a comment so it is clear that this is important. (at least for me it wasn't clear otherwise)